backend/src/main/java/com/bakdata/conquery/apiv1/MeProcessor.java

```diff
@@ -25,7 +25,7 @@ public FEMeInformation getUserInformation(User user){
 		return FEMeInformation.builder()
```

Comment:
Bitte kommentieren, ich möchte in jeder Klasse einen kurzen Umriss was die Aufgabe der Klasse und member ist

Reply:
oki